README.md

```diff
@@ -247,6 +247,7 @@ They are the following:
 
 * `gfx_afterglow`: CRT Tube like glow effect that uses the last image of the previous effec. by @cyriax0
 
+* `gfx_rule90`: [Rule90](https://en.wikipedia.org/wiki/Rule_90) cellular automata. by Matt Venn
```

Comment:
```suggestion
* `gfx_rule90`: [Rule90](https://en.wikipedia.org/wiki/Rule_90) cellular automata. by @mattvenn 
```

Reply:
Actually, I thought @username would be rendered as a link, but it isn't, so this is a purely cosmetic change suggested because so far all contributions were listed with a username (by only one user, however). Your choice if you want to keep or change it :)